.github/workflows/main.yml

```diff
@@ -57,21 +57,12 @@ jobs:
         uses: actions/checkout@v2
 
       - name: Get operator-sdk image
-        run: curl --output operator-sdk -JL https://github.com/operator-framework/operator-sdk/releases/download/$RELEASE_VERSION/operator-sdk-$RELEASE_VERSION-x86_64-linux-gnu
+        run: sudo curl --output /usr/local/bin/operator-sdk -JL "https://github.com/operator-framework/operator-sdk/releases/download/$RELEASE_VERSION/operator-sdk_linux_amd64" && sudo chmod +x /usr/local/bin/operator-sdk
         env:
-          RELEASE_VERSION: v0.19.4
-
-      - name: Make operator-sdk executable
-        run: chmod +x operator-sdk
-
-      - name: Move operator-sdk binary
-        run: sudo mv operator-sdk /usr/local/bin
-
-      - name: Create working directory
-        run: mkdir /tmp/bundle
+          RELEASE_VERSION: v1.16.0
 
       - name: Generate bundle
-        run: WORKING_DIR=/tmp/bundle ./build/generate_bundle.sh
```

Comment:
Removed use of WORKING_DIR here because it's not easily controlled without making changes to the Makefile, so I'm just going to use the default locations. It seems to be working fine in CI so I don't see the need to extend Makefile to allow separate directories.